spring-session-hazelcast/hazelcast4/src/integration-test/java/org/springframework/session/hazelcast/AbstractHazelcast4IndexedSessionRepositoryITests.java

```diff
@@ -198,4 +201,51 @@ void createSessionWithSecurityContextAndFindById() {
 		this.repository.deleteById(sessionId);
 	}
 
+	@Test // gh-1899
```

Comment:
We typically add a comment referencing the GitHub issue when it is not immediately obvious what the purpose of the test is. I think this test is clear enough that we don't have to point to the associated issue.
```suggestion
	@Test
```